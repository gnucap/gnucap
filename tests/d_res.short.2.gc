spice

.param r=0
r1 1 2 {1-r}
r2 1 0 r
v1 2 0 1

.print dc v(nodes)
.dc r 0 1 .5
.dc r 1 0 -.5

.print dc v(nodes) y(r*) iter(0)
.dc r 0 1 .5 loop trace iter

.status notime
.end
