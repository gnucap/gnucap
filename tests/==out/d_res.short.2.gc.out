#           v(1)       v(2)      
r2: short circuit
r2: short circuit
r2: short circuit
r2: short circuit
 0.         9.9998u    0.99999   
 0.5        0.5        0.99999   
r1: short circuit
 1.         0.99998    0.99999   
#           v(1)       v(2)      
r1: short circuit
 1.         0.99998    0.99999   
 0.5        0.5        0.99999   
r2: short circuit
r2: short circuit
r2: short circuit
 0.         9.9998u    0.99999   
#           v(1)       v(2)       y(r1)      y(r2)      iter(0)   
 0.         0.         0.         1.         100.K      0.        
r2: short circuit
-1.         9.9998u    0.99999    1.         100.K      1.        
r2: short circuit
-2.         9.9998u    0.99999    1.         100.K      2.        
r2: short circuit
-3.         9.9998u    0.99999    1.         100.K      3.        
r2: short circuit
 0.         9.9998u    0.99999    1.         100.K      4.        
 0.         9.9998u    0.99999    1.         100.K      0.        
-1.         0.5        0.99999    2.         2.         1.        
-2.         0.5        0.99999    2.         2.         2.        
 0.5        0.5        0.99999    2.         2.         3.        
 0.         0.5        0.99999    2.         2.         0.        
r1: short circuit
-1.         0.99999    0.99999    100.K      1.         1.        
r1: short circuit
-2.         0.99999    0.99999    100.K      1.         2.        
r1: short circuit
 1.         0.99999    0.99999    100.K      1.         3.        
 0.         0.99999    0.99999    100.K      1.         0.        
-1.         0.5        0.99999    2.         2.         1.        
-2.         0.5        0.99999    2.         2.         2.        
 0.5        0.5        0.99999    2.         2.         3.        
 0.         0.5        0.99999    2.         2.         0.        
r2: short circuit
-1.         9.9998u    0.99999    1.         100.K      1.        
r2: short circuit
-2.         9.9998u    0.99999    1.         100.K      2.        
r2: short circuit
 0.         9.9998u    0.99999    1.         100.K      3.        
Gnucap   System status
iterations: op=0, dc=16, tran=0, fourier=0, total=36
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=2, subckt=0, model=0, total=2
dctran density=100.0%, ac density=100.0%
